variables.tf

```diff
@@ -19,13 +19,19 @@ variable "vpc_id" {
   type        = string
 }
 
+variable "subnet_id" {
```

Comment:
I agree to you that the current names of varialbes can be conflicting.

- `subnet_id_runners` is intended as the subnet in which the docker machine instancesa are created
- `subnet_ids_gitlab_runner` is the subnet to host the instance with gitlab agent.

As you mention in the PR this drops the option to deploy to different subnets.  I have no clue if some is doing that. But don't see really a case for that.